bgpd/bgpd.h

```diff
@@ -1757,6 +1757,7 @@ struct peer {
 #define PEER_DOWN_PFX_COUNT             33U /* Reached received prefix count */
 #define PEER_DOWN_SOCKET_ERROR          34U /* Some socket error happened */
 #define PEER_DOWN_RTT_SHUTDOWN          35U /* Automatically shutdown due to RTT */
+#define PEER_DOWN_SUPPRESS_FIB_PENDING	 36U /* Suppress fib pending changed */
```

Comment:
Seems we have a bit unaligned here.

Reply:
this is what clang-format has decided on.  I have no idea why it's not aligned